### DISCUSSION TITLE
NodeJS Support

### DISCUSSION THREAD

**Discussion by zamgill** [bootstrap-email/bootstrap-email#161] [Ideas]
Hi,

I would like to help bring NodeJS support for this project but need some guidance on where / how to being.

- Do I just fork the project and get started with it? 
- Are there any things I should keep in mind?

**Comment by stuyam** [comment]
It's a very good question, I would love to figure out a way to make bootstrap email more universal and be able to be supported in ruby and js environment. It's not entirely trivial but I do think it is possible. Things like the sass stuff would be transferable which is nice, but things like the ruby compiler and converters would need to be rebuilt in js. Or do something like an embedded executable in ruby or js to be supported in other languages. But that adds a lot of file size and  isn't perfect. Open to ideas all around 👍
